packages/java/endpoint/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -4,6 +4,4 @@ com.vaadin.hilla.ApplicationContextProvider
 com.vaadin.hilla.crud.CrudConfiguration
 com.vaadin.hilla.startup.EndpointRegistryInitializer
 com.vaadin.hilla.startup.RouteUnifyingServiceInitListener
-com.vaadin.hilla.route.ClientRouteRegistry
-com.vaadin.hilla.route.RouteUtil
```

Comment:
Not sure if RouteUtil should be removed from the AutoConfiguration imports, IIRC, the some Hilla security code samples related to spring security configuration (user codes) is instructing to utilize this.

Reply:
Route util requires ClientRouteRegistry and as that doesn't exist we do not have a RouteUtil either anymore.
What parts of the functionality is referenced for use? The class itself was even marked as `For internal use only`

Reply:
At least this repo uses RouteUtil:
https://github.com/vaadin/flow-hilla-hybrid-example/blob/v24.4/src/main/java/org/vaadin/example/security/SecurityConfig.java#L30
Let's ping @cromoteca as there's a high chance this have changed very recently and I'm not up-to-date.